engine/src/main/scala/cromwell/webservice/routes/WomtoolRouteSupport.scala

```diff
@@ -0,0 +1,75 @@
+package cromwell.webservice.routes
+
+import akka.actor.{ActorRef, ActorRefFactory}
+import akka.http.scaladsl.marshallers.sprayjson.SprayJsonSupport._
+import akka.http.scaladsl.model._
+import akka.http.scaladsl.server.Directives._
+import akka.http.scaladsl.server.Route
+import akka.pattern.ask
+import akka.stream.ActorMaterializer
+import akka.util.Timeout
+import cromwell.core.WorkflowSourceFilesCollection
+import cromwell.services.womtool.WomtoolServiceMessages.{DescribeFailure, DescribeRequest, DescribeResult, DescribeSuccess}
+import cromwell.services.womtool.WomtoolServiceMessages.JsonSupport.workflowDescriptionFormat
+import cromwell.webservice.WebServiceUtils
+import cromwell.webservice.WebServiceUtils.EnhancedThrowable
+
+import scala.concurrent.ExecutionContext
+import scala.util.{Failure, Success}
+
+trait WomtoolRouteSupport extends WebServiceUtils {
```

Comment:
Thanks for continuing the trend of not putting **everything** in CromwellApiService :) 💯

Reply:
Word, mega-classes are bad news